server/game/player.js

```diff
@@ -616,6 +617,13 @@ class Player extends Spectator {
         this.drawPhaseCards = DrawPhaseCards;
     }
 
+    announceForcedPlotSelection() {
+        //TODO Melee: The forced plot should not be revealed to all players so needs to be whispered
```

Comment:
Hmm, this is a good point. Perhaps the message shouldn't say the plot at all?

Also, I've been trying to move away from putting these types of methods onto the Player class, particularly one like this that doesn't modify the player in anyway. Probably could just be done directly in the new simple step in the plot phase.